react/training-project/src/app/screens/AuthorizedRoute/index.js

```diff
@@ -21,7 +21,7 @@ class AuthorizedRoute extends Component {
         <Topbar logOut={this.handleLogOut} />
         <Switch>
           <Route path={GAME} component={Game} />
-          <Route path={PROFILE} component={Profile} />
+          <Route path={PROFILE} render={() => <Profile {...this.props.userdata} />} />
```

Comment:
What you can do is create a component called AuthorizedRoute that will have the logic to prevent unlogged user see views that require to be logged in.